rest/route/task.go

```diff
@@ -435,6 +436,8 @@ func (rh *taskSetHasCedarResultsHandler) Factory() gimlet.RouteHandler {
 
 func (rh *taskSetHasCedarResultsHandler) Parse(ctx context.Context, r *http.Request) error {
 	rh.taskID = gimlet.GetVars(r)["task_id"]
+	vals := r.URL.Query()
+	rh.failed = vals.Get("failed") == "true"
```

Comment:
I'm pretty sure you can't set GET query parameters on a POST request (I think I've tried this and it doesn't work). You have to send it in the request body because that's what the HTTP spec requires.

Reply:
It worked in staging, I just verified.

Reply:
Hmm, but it's still not preferable to do it like this. Can't you just serialize a JSON payload and stick it in the body? Best practice is to use query parameters for locating resources, not to specify state changes.